build.gradle

```diff
@@ -137,7 +137,7 @@ subprojects {
                 protobuf_plugin: 'com.google.protobuf:protobuf-gradle-plugin:0.7.4',
                 protobuf_util: "com.google.protobuf:protobuf-java-util:${protobufVersion}",
 
-                netty: 'io.netty:netty-codec-http2:4.1.0.CR6',
+                netty: 'io.netty:netty-codec-http2:[4.1.0.CR6]',
                 netty_epoll: 'io.netty:netty-transport-native-epoll:4.1.0.CR6' + epoll_suffix,
```

Comment:
Our libraries aren't using native-epool, so I wasn't very concerned about pinning it, since we'll already be using the correct version. It's only for people that depend on our artifacts.